FAQs/README.md

```diff
@@ -338,3 +341,50 @@ stages:
         azurePowerShellVersion: 'LatestVersion'
 ```
 
+- ### **UI**
+
+### **1. I am not able to see subscriptions in AzTS UI. Are there any prequisites or any settings that I am missing?**
+If you are not able to see subscriptions in AzTS UI , please check below prerequisites followed by troubleshooting steps:
+
+### Prerequisites:
+1. Signed in user must have one of the following permissions at subscription or resource group scope: Owner, Contributor, ServiceAdministrator, CoAdministrator, AccountAdministrator, Security Reader, Security Admin.
+
+2. Subscription scan should have completed for the day. Automated AzTS scans are configured to start at approximately 1:00 AM UTC. You can use the [On-Demand scan](../01-Setup%20and%20getting%20started/README.md#2-manually-trigger-azts-on-demand-scan-for-entire-tenant) command to trigger the scan immediately after the installation.
+
+3. In case access is recently granted, it will reflect in AzTS from next scan onwards. So in case of automated AzTS scan it would take next 24 hours to reflect latest access. You can use the [On-Demand scan](README.md#2-manually-trigger-azts-on-demand-scan-for-entire-tenant) command to trigger the scan in case you want to reflect newly added access immediately.
+
+### **TroubleShooting Steps:**
+
+Please check if below required permissions are granted. These permissions require admin consent. Therefore, user must be a member of one of the following administrator roles: </br>Global Administrator or Privileged Role Administrator.
```

Comment:
Is admin consent required for external users as well or ONLY for Microsoft tenants? If not, we might want to prefix this question with [For Microsoft Only].

Reply:
Admin consent will be required for external users as well based on tenant level settings.  There can be multiple ways Admin can consent like by executing PS script/form based but consent would be needed.